frontend/src/NavItems/NavItems.tsx

```diff
@@ -0,0 +1,30 @@
+import { useNavigate } from "react-router-dom"
+
+const NavItems: React.FC = () => {
```

Comment:
I think calling these `NavItems` might be a bit vague, I don't find it clear to know what this component is doing. How would you feel about changing it to something like `Navigation` or `NavigationLinks`?

Reply:
We'll update it to NavigationLinks, thank you!